frontend/src/components/auth/ChangePasswordForm.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Button,
+  Box,
+  Input,
+  Cell,
+  Row,
+  Col,
+  Container,
+  Card,
+  FormField,
+} from 'wix-style-react';
+import ErrorNotification from '../ErrorNotification';
+
+const ChangePasswordForm = ({ onChange }) => {
+  const [oldPassword, setOldPassword] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmedPassword, setConfirmedPassword] = useState('');
+  const [showNotification, setShowNotification] = useState(false);
+
+  useEffect(() => {
+    const listener = event => {
+      if (event.code === 'Enter' || event.code === 'NumpadEnter') {
+        if (oldPassword !== '' && newPassword !== '' && confirmedPassword !== '')
```

Comment:
why is validation here, and not in `handleChangePasswordBtn` ?

Reply:
I moved it to handleBtn now